docs/Documentation/Development/Programming_Models/gpu_hpc.md

```diff
@@ -70,7 +70,6 @@ The following are some GPU-relevant environment variables you can set in your su
 | :--                    | :--         |
 | `SLURM_GPUS_ON_NODE`   | Quantity of GPU devices available to a Slurm job. Set by Slurm. |
 | `SLURM_JOB_GPUS`       | GPU device ID(s) available to a Slurm job. Starts with `0`. Set by Slurm. |
```

Comment:
I think the SLURM_JOB_GPUS definition should be clarified too, since it doesn't always start with 0. 
Maybe something like: 
The global GPU device ID(s) of the GPUs allocated to a job. Set by Slurm.

Reply:
Makes sense; just pasted in the SchedMD definitions and linked to them as well. Also added a note about 'nvidia-smi -L'